### PR TITLE
TImeline Replies Test

### PR DESCRIPTION
**Pull request by pdb493**

### CODE REVIEW

File: app/Http/Kernel.php [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 namespace App\Http;
 
+// TEST COMMENT AGAIN
```
[the remark this answers is not in the document]
Comment (pdb493) [reply app/Http/Kernel.php:5]: inline reply 1

**Review by pdb493 — COMMENTED** [review]
summary comment 1 (customer)

**Comment by pdb493** [comment]
> summary comment 1 (admin reviewer)
> 
> Image of Chanse T **Chanse T**
> 
> Was this helpful? [Yes](https://patrick-9a73.pullrequest.tech/reviews/ratings/bdac8eed-73bd-42a1-ba2d-cf4eca1d8cf5/true) | [No](https://patrick-9a73.pullrequest.tech/reviews/ratings/bdac8eed-73bd-42a1-ba2d-cf4eca1d8cf5/false)
> 
> > _Reviewers will be notified any time you reply to their comments or commit new changes._
> > _You can also request a full follow-up review from your [PullRequest dashboard](https://patrick-9a73.pullrequest.tech/dash/projects/1/pdb493/test/reviews/)._

test quote reply 1

File: app/User.php [hunk 2]
```diff
@@ -1,8 +1,9 @@
 <?php
+
 namespace App;
 
 use Illuminate\Notifications\Notifiable;
-use Illuminate\Foundation\Auth\User as Authenticatable;
```
[the remark this answers is not in the document]
Comment (pbarsallo) [reply app/User.php:5]: test inline reply 2

File: app/Http/Kernel.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (pbarsallo) [reply app/Http/Kernel.php:5]: test inline reply 3 (review)

**Review by pbarsallo — COMMENTED** [review]
test review

File: app/User.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (pdb493) [reply app/User.php:5]: another inline reply

**Pull request closed without merging**